### DISCUSSION TITLE
The recent xLSTM paper

### DISCUSSION THREAD

**Discussion by JensRoland** [BlinkDL/RWKV-LM#245] [Ideas]
I assume you have already seen this, but in case you haven't: there is a new model closely related to RWKV, which the authors have named xLSTM. The paper (linked below) makes several references to RWKV and claims to benchmark favorably against it.

There doesn't seem to be any published code with the paper, but the paper is quite interesting in itself: https://arxiv.org/abs/2405.04517

**Comment by RillmentGames** [comment]
Code got released: [https://github.com/NX-AI/xlstm](https://github.com/NX-AI/xlstm)
